### DISCUSSION TITLE
Does the MkDocs project have an official logo/icon?

### DISCUSSION THREAD

**Discussion by shollyethan** [mkdocs/mkdocs#3854] [General]
I'm the maintainer of [selfh.st/icons](https://selfh.st/icons) and would love to add something for MkDocs, but can't seem to find an icon or logo online.

Is there something available that I can use? Thanks!

**Comment by arregit** [comment]
According to [wikipedia](https://commons.wikimedia.org/wiki/File:MkDocs_Logo.png) that is the official logo.
I would love to have a svg version of the logo, to be able to properlz reference this project.

**Reply by shollyethan** [reply]
I was actually able to recreate this by hand -- you can find it in my repo/collection via the links below:

- https://github.com/selfhst/icons/blob/main/svg/mkdocs.svg
- https://cdn.jsdelivr.net/gh/selfhst/icons/svg/mkdocs.svg
